Task 2/Task 2/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Дана длина ребра куба. Найти его объем и длину диагонали.
+            Console.WriteLine("Введите длину ребра куба");
+            double rebro = double.Parse(Console.ReadLine());
+            double objem = Math.Pow(rebro, 3);
+            double diagonal = Math.Sqrt(3) * (rebro);
+            Console.WriteLine($"объем = {objem}");
```

Comment:
Старайтесь для имен переменных использовать английские слова, а не русские транслитом. Транслит затрудняет чтение кода
```suggestion
            double edge = double.Parse(Console.ReadLine());
            double volume = Math.Pow(edge, 3);
            double diagonal = Math.Sqrt(3) * (edge);
            Console.WriteLine($"объем = {volume}");
```